api/token.go

```diff
@@ -517,10 +519,21 @@ func (a *API) issueRefreshToken(ctx context.Context, conn *storage.Connection, u
 	}, nil
 }
 
-func (a *API) setCookieToken(config *conf.Configuration, tokenString string, session bool, w http.ResponseWriter) error {
+// setCookieTokens sets the access_token & refresh_token in the cookies
+func (a *API) setCookieTokens(config *conf.Configuration, token *AccessTokenResponse, session bool, w http.ResponseWriter) error {
+	_ = a.setCookieToken(config, "access_token", token.Token, session, w)
+	_ = a.setCookieToken(config, "refresh_token", token.RefreshToken, session, w)
+	return nil
+}
+
+func (a *API) setCookieToken(config *conf.Configuration, name string, tokenString string, session bool, w http.ResponseWriter) error {
+	cookieName := config.Cookie.Key
+	if name != "" {
+		cookieName += "_" + name
+	}
 	exp := time.Second * time.Duration(config.Cookie.Duration)
 	cookie := &http.Cookie{
-		Name:     config.Cookie.Key,
+		Name:     cookieName,
 		Value:    tokenString,
 		Secure:   true,
```

Comment:
Something to note and test is that

> Insecure sites (http:) cannot set cookies with the Secure attribute (since Chrome 52 and Firefox 52). For Firefox, the https: requirements are ignored when the Secure attribute is set by localhost (since Firefox 75).

So, will want to test what the cookie is in local testing with just http.

Reply:
It's possible on localhost: https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#restrict_access_to_cookies
![image](https://user-images.githubusercontent.com/5748289/149292093-dcffb42e-9fbe-4014-9813-97ac9c164b65.png)